src/github.com/getlantern/flashlight/util/http.go

```diff
@@ -2,21 +2,138 @@ package util
 
 import (
 	"crypto/tls"
+	"errors"
 	"fmt"
 	"net"
 	"net/http"
 	"net/url"
+	"sync/atomic"
 	"time"
 
+	"github.com/getlantern/fronted"
 	"github.com/getlantern/golog"
 	"github.com/getlantern/keyman"
 	"github.com/getlantern/waitforserver"
 )
 
+const (
+	defaultAddr = "127.0.0.1:8787"
```

Comment:
It's just hard for me to imagine a scenario where we'd actually ever change that using the cloud config, and supporting that option complicates the code quite a bit because then anything using an HTTP client to the localhost has to reconfigure with new configs.